### PR TITLE
v1.1.0-Release

### PR DESCRIPTION
**Pull request by CarelessOutcast**

### CODE REVIEW

File: _frontend/LICENSE.md
Comment (CarelessOutcast) [inline _frontend/LICENSE.md:1]: Change Copyright

**Review by CarelessOutcast — COMMENTED** [review]
Everylooks good!

**Comment by Aleizana** [comment]
The website still worked when I made the changes to my branch before Nick merged, so I'm going to assume that it was only the unit tests that weren't working before the last merge. As long as it worked with the migrations, we're solid.

**Review by Auqtruong — APPROVED** [review]
Looks good, just need to add the contributor section for frontend readme

**Review by Aleizana — APPROVED** [review]

**Review by Aleizana — APPROVED** [review]

**Pull request merged**